Bottom: da2a09e7dd25ff1f088789191568f7c967202645
Top:    028858e444b236d9cca834ed6fc99388a87c7be3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:20:20 +0000

Debug on


---

diff --git a/src/polygon.h b/src/polygon.h
index a02fd99..a369096 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8//40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 /* adjustment to make the segments outline the circle rather than connect
